qiskit/visualization/pass_manager_visualization.py

```diff
@@ -179,17 +279,3 @@ def pass_manager_drawer(pass_manager, filename=None, style=None, raw=False):
         if filename:
             image.save(filename, "PNG")
         return image
-
-
-def _get_node_color(pss, style):
```

Comment:
note: this is still there on line 101 or thereabouts, git diff just makes it look like a deletion / readdition